src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const UnauthenticatedContent = () => (
+  <div className="vads-u-text-align--center">
+    <h2 className="vads-u-font-size--h3">We can’t load this page</h2>
+    <div className="vads-u-font-size--lg">
+      Something went wrong on our end. Please double check the URL and make sure
+      you are signed in.
+    </div>
+  </div>
```

Comment:
I feel like all these error messages should be inside a `va-alert`. I haven't seen the design, but I feel like it makes sense.

Reply:
I personally agree with you, but this is the design that was approved:
https://www.sketch.com/s/8b025901-295d-4929-95cc-8dbeb28454b3/a/dlL1L52

Reply:
![Screen Shot 2023-01-04 at 12 03 26 PM](https://user-images.githubusercontent.com/13838621/210620299-50131008-1416-4b58-8796-99b328e5bf4f.png)